webapp/pages/profile/_id/_slug.vue

```diff
@@ -67,7 +69,7 @@
           <ds-space margin="small">
             <template v-if="!myProfile">
               <hc-follow-button
-                v-if="!user.isMuted"
+                v-if="!user.isMuted || !user.isBlocked"
```

Comment:
this is questionable what we want to do here cause it was the case that when a user `BLOCKED` another, that it would remove the `Follow` button and replace it with a `Unblock user` button
what should happen now? If a user has `MUTED` another user, it should appear `Unmute user`... and if they have `BLOCKED` as well?
should there be two buttons? I guess that might be noisy...